mandatory/1-syntax-errors.js

```diff
@@ -1,16 +1,17 @@
 // There are syntax errors in this code - can you fix it to pass the tests?
 
-function addNumbers(a b c) {
+function addNumbers(a, b, c) {
   return a + b + c;
 }
 
-function introduceMe(name, age)
-  return "Hello, my name is " + name "and I am " age + "years old";
+function introduceMe(name, age) {
+  return "Hello, my name is " + name + " and I am " + age + " years old";
+}
+  
 
 function getTotal(a, b) {
-  total = a ++ b;
-
-  return "The total is total";
+  total = a + b;
```

Comment:
This variable has been initialized without it being defined first.
```suggestion
  const total = a + b;
```